### ISSUE TITLE
Port passed to pg:pull is ignored

### ISSUE THREAD

**Issue by adamrainsby** [heroku/heroku-pg#68]
I raised the same issue here https://github.com/heroku/cli/issues/357 but I guess this is where the issue actually is.

Since Friday whenever I run 

```
PGPORT=5433 heroku pg:pull ...
```

I get

```
createdb: could not connect to database template1: could not connect to server: Connection refused
	Is the server running on host "localhost" (::1) and accepting
	TCP/IP connections on port 5432?
could not connect to server: Connection refused
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 5432?
```

I have postgres running on port 5433 not 5432 so obviously the connection is refused. 

I also noticed there aren't any tests for non-standard ports in test/commands/pull.js

**Comment by ransombriggs** [comment]
@adamrainsby I just shipped a new version (5.5.10) you can get by running `heroku update`.  Please let me know if that fixes this bug.  fwiw, the PR with the fix is here https://github.com/heroku/heroku-pg/pull/70/files

**Comment by adamrainsby** [comment]
Perfect, thanks.
